.github/workflows/static-scan.yml

```diff
@@ -28,22 +28,31 @@ jobs:
     - uses: brpaz/hadolint-action@v1.2.1
       name: Run Hadolint
       with:
-       dockerfile: Dockerfile
+       dockerfile: ./docker/userspacecni/Dockerfile
```

Comment:
may want to consider a non relative path on a future push

Reply:
This PR is a WIP for now, so this will change later